### PR TITLE
Home Page

### LINKED ISSUE #38: Game has home page for users to navigate from

### PR DESCRIPTION
**Pull request by Cynthiack**
closes #38

### CODE REVIEW

File: guessleHome.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+
```
Comment (Keowa-1877616) [inline guessleHome.js:1]: Still requires integration with other Html files
Requires singleplayer button to play singleplayer game
Requires multiplayer button to play multiplayer game (Needs to ask whish multiplayer mode will be played)
Requires a log out button
Requires display that shows user name
Requires display of user's game stats

File: homePage.html [hunk 2]
```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
```
Comment (Keowa-1877616) [inline homePage.html:1]: Still requires integration with other Html files
Requires singleplayer button to play singleplayer game
Requires multiplayer button to play multiplayer game (Needs to ask whish multiplayer mode will be played)
Requires a log out button
Requires display that shows user name
Requires display of user's game stats

**Review by Keowa-1877616 — COMMENTED** [review]
Functionality still required

**Review by TTRasemana — CHANGES_REQUESTED** [review]
Agreed with @Keowa-1877616 for the functionality to be  improved. Hence see previous review.

**Review by HyChriFo — APPROVED** [review]
The buttons are working as they should

**Pull request merged**